### PR TITLE
Cleanup pull request 6

### PR DESCRIPTION
**Pull request by RowenStipe**
Cleaned up https://github.com/Tox/Sounds/pull/6 according to README
guidelines.

Also added converted filenames to README for pack for the pack author and documentation.

### CODE REVIEW

File: Modern Sonar/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Previous file names
+*  call_incomming > IncommingAudio
+*  call_ending > CallDrop
+*  contact_Online > ContactOnline
+*  contact_statusChange > Group/NameChange
+*  contact_StatusChange_Offline > ContactOffline
+*  contact_StatusChange_Online > Invite
+*  filetran._fail > Failure
+*  filetran._complete > Sucess
+*  filetran._incomming > Incomming
```
Comment (Zer0-One) [inline Modern Sonar/README.md]: Spelling error

File: Modern Sonar/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Previous file names
+*  call_incomming > IncommingAudio
+*  call_ending > CallDrop
+*  contact_Online > ContactOnline
+*  contact_statusChange > Group/NameChange
+*  contact_StatusChange_Offline > ContactOffline
+*  contact_StatusChange_Online > Invite
+*  filetran._fail > Failure
+*  filetran._complete > Sucess
```
Comment (Zer0-One) [inline Modern Sonar/README.md]: spelling error

File: Modern Sonar/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# About Modern Sonar
+Modern Sonar is a sound pack made by Nowologix.
+
+Previous file names
+*  call_incomming > IncommingAudio
```
Comment (Zer0-One) [inline Modern Sonar/README.md]: Spelling error

**Comment by Zer0-One** [comment]
Aside from the spelling errors in the README, a few of the filenames are incorrect as well

**Comment by RowenStipe** [comment]
@__@ I have no one to blame but my own incompetence...

**Comment by Zer0-One** [comment]
@RowenStipe PSYYYDUCK

**Comment by Zer0-One** [comment]
@RowenStipe There's no license. I can't merge this until @nowologix gives this a license.

**Comment by RowenStipe** [comment]
@Zer0-One Yeah I noticed that, there was none given by @nowologix in his inital PR so I guess we wait to get a licence from him.

**Comment by Zer0-One** [comment]
@nowologix still interested in contributing?

**Comment by Zer0-One** [comment]
RIP

**Pull request closed without merging**